warehouse/accounts/forms.py

```diff
@@ -84,13 +84,13 @@ def validate_password(self, field):
             try:
                 if not self.user_service.check_password(userid, field.data):
                     raise wtforms.validators.ValidationError(
-                        "The password you have provided is invalid. Please "
-                        "try again."
+                        "The password you have provided is invalid. "
+                        "Try again."
                     )
             except TooManyFailedLogins:
                 raise wtforms.validators.ValidationError(
                     "There have been too many unsuccessful login attempts, "
-                    "please try again later."
+                    "Try again later."
```

Comment:
We either need to put a period at the end of the previous phrase, or lowercase this `Try`.